src/main/java/io/bloombox/schema/partner/PartnerScope.java

```diff
@@ -1,70 +1,78 @@
-/*
- * Copyright 2018, Bloombox, LLC.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
 // Generated by the protocol buffer compiler.  DO NOT EDIT!
 // source: partner/PartnerScope.proto
 
 package io.bloombox.schema.partner;
 
 /**
+ * <pre>
+ * Specifies authentication scopes at the partner level.
+ * </pre>
+ *
  * Protobuf enum {@code bloombox.schema.partner.PartnerScope}
  */
 public enum PartnerScope
     implements com.google.protobuf.ProtocolMessageEnum {
   /**
    * <pre>
-   * - Basic
+   * Basic authentication as a member of a partner. This includes partner staff, managers, owners, and so on.
    * </pre>
    *
    * <code>AUTH = 0;</code>
    */
   AUTH(0),
   /**
+   * <pre>
+   * Authorizes the sharing of contact information related to a partner account.
+   * </pre>
+   *
    * <code>CONTACT = 1;</code>
    */
   CONTACT(1),
   /**
+   * <pre>
+   * Authorizes access to the Dashboard for a given partner account.
+   * </pre>
+   *
    * <code>DASHBOARD = 2;</code>
    */
   DASHBOARD(2),
   /**
    * <pre>
-   * - Menus
+   * Authorizes access to read menu data for a given partner account.
    * </pre>
    *
    * <code>MENU_DATA_READ = 10;</code>
    */
   MENU_DATA_READ(10),
```

Comment:
I assume we jump from **2** to **10** to allow for other types of product accesses in the future

Reply:
exactly - in case we need some other core ones between 2 and 10.